Makefile

```diff
@@ -2,13 +2,15 @@
 
 FLAGS=
 
-CHAPTERS=$(patsubst book/%.md,%,$(wildcard book/*.md))
+CHAPTERS=about preface intro history http graphics text html \
```

Comment:
This ensures the chapters are in the correct order, which is necessary for latex output.